Website/src/rendering/src/stories/PageContent/FullImageSection.stories.tsx

```diff
@@ -14,7 +14,7 @@ export const Left = Template.bind({});
 Left.args = {
   fields: {
     cssClass: {
-      value: 'section__full-image--offer',
+      value: 'section__full-image',
```

Comment:
As with the serialized items, this class is not needed now. It can be an empty string.